IBMigrationTool/migrate.sh

```diff
@@ -0,0 +1,14 @@
+
+bin_path="IBMigrationTool"
+migrationtool_binary_download_url="https://raw.githubusercontent.com/3sidedcube/ThunderBasics/feature/migration_tool_improvements/IBMigrationTool/IBMigrationTool"
```

Comment:
Just remember this branch might be deleted in which case we'd need to switch to `develop` or something